CHANGELOG.md

```diff
@@ -1,5 +1,45 @@
 # Azure Kubernetes Service Changelog
 
+## Release 2024-03-31
+
+Monitor the release status by regions at [AKS-Release-Tracker](https://releases.aks.azure.com/).
+
+### Announcements
+* Force migrating all Image Cleaner clusters to addon v2.
+
+### Release Notes
+* Features:
+  * Created ZRS disk by default in built-in storage classes on a multi-zone AKS cluster
+
+* Component Updates: 
```

Comment:
We generally write it as "Upgrade foo from vX.X.X to vY.Y.Y" and link each to change logs. 
Please review the past release notes: https://github.com/Azure/AKS/pull/4174/files

Reply:
Fixed, thank you